docs/tutorials/intro_tutorial.ipynb

```diff
@@ -60,7 +60,7 @@
    "source": [
     "### Tutorial Setup\n",
     "\n",
-    "Create and activate a [virtual environment](http://docs.python-guide.org/en/latest/dev/virtualenvs/). *Python version 3.8 or higher is required*.\n",
+    "Create and activate a [virtual environment](http://docs.python-guide.org/en/latest/dev/virtualenvs/). *Python version 3.9 or higher is required*.\n",
```

Comment:
Do we still want to do 3.9 as it seems Solara only works with 3.10 or higher?

Reply:
Quite the opposite, solara still supports Python 3.6. They are quite proud of it, although i cant find the source for that claim right now. But their unit tests still run on Python 3.6 https://github.com/widgetti/solara/blob/master/.github/workflows/unittest.yml

Reply:
Following [SPEC 0](https://scientific-python.org/specs/spec-0000/), the successor of [NEP 29](https://numpy.org/neps/nep-0029-deprecation_policy.html), Python 3.9 can also be dropped at this point.

That would allow us to start using Structural Pattern Matching (also in examples) and use new typing features. See [What’s New In Python 3.10](https://docs.python.org/3/whatsnew/3.10.html).

Users still on Python 3.8 and 3.9 can of course still keep using the current Mesa versions, as always.